.vsts-ci-templates/build.yml

```diff
@@ -1,27 +1,21 @@
 parameters:
   cmake_args: ''
-  artifact_name: ''
 
 steps:
+- script: |
+    cat /proc/cpuinfo
+  displayName: /proc/cpuinfo
+  condition: succeededOrFailed()
```

Comment:
Can this actually fail?

Reply:
Probably not, but I think we want it to run on every build regardless of what else has failed, even if it gets copy-and-pasted to a later step.

Reply:
Ah good point